apps/nextjs-website/src/pages/index.tsx

```diff
@@ -1,3 +1,21 @@
-export default function Home() {
-  return <>Home Page</>;
-}
+import Layout, { LayoutProps } from '@/components/organisms/Layout/Layout';
+import { getProducts } from '@/lib/api';
+import { GetStaticProps, GetStaticPropsResult } from 'next';
+
+type HomeProps = LayoutProps;
+
+export const getStaticProps: GetStaticProps<
+  HomeProps
+> = (): GetStaticPropsResult<LayoutProps> => {
+  return { props: { products: getProducts().concat() } };
```

Comment:
```suggestion
  return { props: { products: [...getProducts()] } };
```
I think adding `.concat()` could mislead to think that we need to combine two different arrays.